elastic-buffer-sim/exe/Main.hs

```diff
@@ -504,12 +506,31 @@ optionParser =
               )
           )
     <*> option auto
-          (  long "offsets"
+          (  long "clock-offsets"
           <> short 't'
           <> metavar "NUM LIST"
           <> value []
           <> showDefault
-          <> help "Initital clock offsets (randomly generated if missing"
+          <> help "Initital clock offsets (randomly generated if missing)"
+          )
+    <*> option auto
+          (  long "startup-offsets"
+          <> short 'T'
+          <> metavar "NUM LIST"
+          <> value []
+          <> showDefault
+          <> help "Initital startup offsets (randomly generated if missing)"
```

Comment:
Could you add some units to this? I'm guessing this is defined in `fs` or `ps`?

Reply:
It's currently in clock cycles (according to the node's local clock). It was the easiest to setup. Is that ok for you or do you want to have it converted?

Reply:
Yeah that's fine, as long as that unit is mentioned